docs/components/motor/28byj-48.md

```diff
@@ -17,17 +17,14 @@ Viam also supports:
 
 {{< /alert >}}
 
-## Configuration
-
-To configure a `28byj48` motor as a component of your robot, first configure the [board](/components/board/) to which the motor driver is wired.
-
-Then assign your motor a `name` (to identify the motor), `type` (`motor`) and `model` (`28byj48`), and fill in the [attributes](#required-attributes) that apply to your particular hardware.
-Refer to your motor and motor driver data sheets for specifics.

```

Comment:
I think we should also keep that

Reply:
Added 22 & 25 back in on top and attributes, respectively-- 24 is covered